addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -51,66 +55,64 @@ options.registry.mailing_list_subscribe = mass_mailing_common.extend({
-                model: 'mail.mass_mailing.list',
-                method: 'read',
-                args: [[parseInt(mailing_list_id)], ['popup_content'], self.options.recordInfo.context],
-            }).then(function (data) {
-                self.$target.find(".o_popup_content_dev").empty();
-                if (data && data[0].popup_content) {
-                    $(data[0].popup_content).appendTo(self.$target.find(".o_popup_content_dev"));
-                }
-            });
-        });
-    },
-});
 
-WysiwygMultizone.include({
-    events: _.extend({}, WysiwygMultizone.prototype.events, {
-        'click #edit_dialog': 'edit_dialog',
-        'click .o_popup_modal_content [data-dismiss="modal"]': 'close_dialog',
-    }),
-    save: function () {
-        var $target = $('#wrapwrap').find('#o_newsletter_popup');
-        if ($target && $target.length) {
-            this.close_dialog();
-            $('.o_popup_bounce_small').show();
-            if (!$target.find('.o_popup_content_dev').length) {
-                $target.find('.o_popup_modal_body').prepend($('<div class="o_popup_content_dev" data-oe-placeholder="' + _t("Type Here ...") + '"></div>'));
-            }
-            var content = $('#wrapwrap .o_popup_content_dev').html();
-            var newsletter_id = $target.parent().attr('data-list-id');
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'write',
-                args: [
-                    parseInt(newsletter_id),
-                    {'popup_content':content},
-                    this.options.recordInfo.context,
-                ],
-            });
-        }
+        this.$target.on('shown.bs.modal', function () {
+            self.trigger_up('deactivate_snippet');
+        });
         return this._super.apply(this, arguments);
     },
-    destroy: function () {
-        this.close_dialog();
-        this._super();
+    /**
+     * @override
+     */
+    cleanForSave: function () {
+        this.$('.o_newsletter_modal').modal('hide');
     },
 
     //--------------------------------------------------------------------------
-    // Handler
+    // Options
     //--------------------------------------------------------------------------
 
-    close_dialog: function () {
-        $('#wrapwrap').find('#o_newsletter_popup').modal('hide');
+    /**
+     * @override
+     */
+    select_mailing_list: function (previewMode, value) {
+        var self = this;
+        return this._super(previewMode, value).then(function () {
+            self.$target.attr('quick-open', true);
+            self._refreshPublicWidgets();
+        });
     },
-    edit_dialog: function (ev) {
-        $('#wrapwrap').find('#o_newsletter_popup').modal('show');
-        $('.o_popup_bounce_small').hide();
-        $('.modal-backdrop').css("z-index", "0");
+});
+
+WysiwygMultizone.include({
+    _saveEditable: function (editable) {
```

Comment:
Override `_saveElement` instead of `_saveEditable`

Reply:
@qsm-odoo I use `_saveEditable` because we need to remove popup before `_saveElement`. if we Override `_saveElement` then we have to remove popup from editable and `outerHTML` (which is a string and will be saved to `ir.ui.view`).

Reply:
Mmh ok, in fact I am not sure this is a good idea to modify the DOM during save. I think the popup should be removed earlier (in the `cleanForSave` of the option? Or somewhere like that) and the DOM of the popup can be saved as `.data()` on the related snippets for the save to use. Is this possible to do? Thanks

Reply:
Yes we can handle it this way 
Thanks :)